pkgs/applications/audio/deadbeef/plugins/musical-spectrum.nix

```diff
@@ -0,0 +1,38 @@
+{ deadbeef
+, fetchFromGitHub
+, fftw
+, glib
+, gtk3
+, lib
+, pkg-config
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "deadbeef-musical-spectrum-plugin";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "cboxdoerfer";
+    repo = "ddb_musical_spectrum";
+    rev = "a97fd4e1168509911ab43ba32d815b5489000a06";
+    sha256 = "0p33wyqi27y0q1mvjv5nn6l3vvqlg6b8yd6k2l07bax670bl0q3g";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ deadbeef fftw glib gtk3 ];
+  makeFlags = [ "gtk3" ];
+
+  meta = with lib; {
```

Comment:
We prefer having `meta` as the last attribute, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.